to_data_library/data/transfer.py

```diff
@@ -385,8 +400,10 @@ def _get_keys_in_s3_bucket(self, aws_session, bucket_name, prefix_name, wildcard
         regex = re.compile(wildcard)
 
         pages = paginator.paginate(Bucket=bucket_name, Prefix=prefix_name)
-        print(f'pages: {pages}')
+        print(f'Pages: {pages}')
```

Comment:
is this for testing or logging? if loggin then can you use logger?

Reply:
testing - I'll remove